tidypolars/tibble.py

```diff
@@ -61,24 +64,24 @@ def __str__(self):
         df = self.to_polars()
         return df.__str__()
 
-    def __getattribute__(self, attr):
-        if attr in _polars_methods:
-            raise AttributeError
-        return pl.DataFrame.__getattribute__(self, attr)
-
-    def __dir__(self):
-        _tidypolars_methods = [
-            'arrange', 'bind_cols', 'bind_rows', 'colnames', 'clone', 'count',
-            'distinct', 'drop', 'drop_null', 'head', 'fill', 'filter',
-            'inner_join', 'left_join', 'mutate', 'names', 'nrow', 'ncol',
-            'full_join', 'pivot_longer', 'pivot_wider',
-            'pull', 'relocate', 'rename', 'replace_null', 'select',
-            'separate', 'set_names',
-            'slice', 'slice_head', 'slice_tail', 'summarize', 'tail',
-            'to_pandas', 'to_polars', 'write_csv', 'write_parquet'
-        ]
-        return _tidypolars_methods
-
+    # def __getattribute__(self, attr):
```

Comment:
These need to be uncommented. Using a `polars` method on a `Tibble` returns a `polars` DataFrame - meaning `Tibble` methods will no longer work on the returned object. This would be pretty unexpected for the user.

```python
import tidypolars as tp
from tidypolars import col

df = tp.Tibble(dict(x = range(3), y = ["a", "a", "b"]))

type(df.with_columns(col.x * 2))
```

```
polars.dataframe.frame.DataFrame
```

Reply:
Agree